tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
I reverted because white space considerations seem outside the scope of this rule. When running `--fix` the linter will first find the warning for `self-closing-comp` and fix it. The linter will then be run again against the code with fixed `self-closing-comp` rule, and apply the fix for `jsx-space-before-closing`. So it seems that white space and closing tags are unrelated, and this way the fixes for the two rules work together to give the result of producing a self-closing tag with the appropriate white space.